### PR TITLE
Add Filecoin deprecation notice.

### PR DESCRIPTION
**Pull request by bgravenorst**
Add a deprecation notice for Filecoin.

Fixes: #203

### CODE REVIEW

**Review by jframe — APPROVED** [review]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ slug: /
 
 # Web3Signer
 
+:::caution "Filecoin deprecation notice"
```
Comment (alexandratran) [inline docs/index.md:10]:
```suggestion
:::caution Filecoin deprecation notice
```

File: docs/reference/api/filecoin.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ toc_max_heading_level: 2
 
 # Filecoin JSON-RPC API
 
+:::caution "Filecoin deprecation notice"
```
Comment (alexandratran) [inline docs/reference/api/filecoin.md:9]:
```suggestion
:::caution Filecoin deprecation notice
```

**Review by alexandratran — APPROVED** [review]
Just need to remove the quotes.

**Pull request merged**